tests/functional/apps/eos_transaction_builder.py

```diff
@@ -149,7 +154,9 @@ def encode_action_parameters(self, data):
             parameters += pack(f'{len(memo)}s', data['memo'].encode())
 
         return parameters
-
+        
+class SwapToAction(TransferAction):
+    pass
```

Comment:
needs to be implemented, right?

Reply:
I could use your advice here. Maybe drop the SwapToAction class?

`SwapToAction` has the same ABI and `TransferAction`. So I just copied the class definition into a new name. Line 185 is python code to have `SwapToAction` inherit from `TransferAction`. The pass is required by python otherwise a dummy method would be used.

Reply:
Oh I missed the inheritance, probably all good then!